src/goImport.ts

```diff
@@ -20,10 +20,18 @@ export async function listPackages(excludeImportedPkgs: boolean = false): Promis
 		? await getImports(vscode.window.activeTextEditor.document)
 		: [];
 	const pkgMap = await getImportablePackages(vscode.window.activeTextEditor.document.fileName, true);
-
+	const stdLib = await getStandardLibraryPackages();
```

Comment:
Instead of making another call to `go list` which we know to be expensive, why not update `getImportablePackages`  itself return the information on whether a package is standard or not?

Right now, we use `{{.ImportPath}}` and `{{.Dir}}` when calling `go list` in the `goPackages.ts` file.
We can add `{{.Standard}}` to the same call and get a boolean representing what we need

Reply:
If I am not wrong I guess we are using [gopkgs](https://github.com/uudashr/gopkgs) to list the packages and that doesn't support the `{{.Standard}}` flag. That's the reason I wrote another function that uses `go list`.

Reply:
Ah! fair point.

In that case, I would prefer if we can update `gopkgs` to also support `{{.Standard}}`
I am guessing that whatever it does under the hood to fetch the packages, it already knows if a package is standard or not and exposing this info is a valid feature request.

Elsewhere in this project in [goSuggest.ts](https://github.com/microsoft/vscode-go/blob/0.11.7/src/goSuggest.ts#L552-L553) file, a use a very crude way to identify the standard packages which is seeing if the import path contains a ".". 

So, maybe until `gopkgs` is fixed, we can use a similar approach?

Reply:
I will create a PR to support`{{.Standard}}` in `gopkgs` and then continue with this PR. That seems to be the cleanest approach right?

Reply:
@ramya-rao-a  fyi uudashr/gopkgs#17

Reply:
I see that you have used GOROOT in the Dir to determine if the package is standard or not.
We do have access to the {{.Dir}}, so maybe we can use that in the extension, compare with `process.env['GOROOT']`?